ecosystem/sep-0006.md

```diff
@@ -33,11 +34,14 @@ To support this protocol an anchor acts as a server and implements the specified
 
 * An anchor must define the location of their `TRANSFER_SERVER` in their [`stellar.toml`](sep-0001.md). This is how a wallet knows where to find the anchor's server.
 * Anchors and clients may support [SEP-10](sep-0010.md) web authentication to enable authenticated deposits, withdrawals, or transaction history lookups.
+* If the Anchor supports [SEP-38], `ANCHOR_QUOTE_SERVER` must be defined in their [`stellar.toml`](sep-0001.md).
 
 ## API Endpoints
 
 * [`GET /deposit`](#deposit): required
+* [`GET /deposit-exchange`](#deposit-exchange): optional
 * [`GET /withdraw`](#withdraw): required
+* [`GET /withdraw-exchange`](#withdraw-exchange): optional
```

Comment:
💡 This is non-critical, and is a suggestion that could easily be subjective. I think it is useful to group related items together. The deposit endpoints are related, but the fact the exchange endpoints are implemented together and optional and serve the same category of transaction type, I think their affiliation is stronger.
```suggestion
* [`GET /deposit`](#deposit): required
* [`GET /withdraw`](#withdraw): required
* [`GET /deposit-exchange`](#deposit-exchange): optional
* [`GET /withdraw-exchange`](#withdraw-exchange): optional
```

Reply:
https://github.com/stellar/stellar-protocol/pull/973/commits/b6b2a59a097c557bdf52a68c0cde5711be5da54c